src/evaluator/fallbackTreatmentsCalculator/index.ts

```diff
@@ -0,0 +1,49 @@
+import { FallbackTreatmentConfiguration, FallbackTreatment, IFallbackTreatmentsCalculator} from '../../../types/splitio';
+
+export class FallbackTreatmentsCalculator implements IFallbackTreatmentsCalculator {
+  private readonly labelPrefix = 'fallback - ';
+  private readonly control = 'CONTROL';
+  private readonly fallbacks: FallbackTreatmentConfiguration;
+
+  constructor(fallbacks: FallbackTreatmentConfiguration) {
+    this.fallbacks = fallbacks;
+  }
+
+  resolve(flagName: string, label?: string | undefined): FallbackTreatment {
```

Comment:
Undefined is redundant with `?`
```suggestion
  resolve(flagName: string, label?: string): FallbackTreatment {
```